src/js/vre/chapter31/components/IntroductionPage.jsx

```diff
@@ -2,28 +2,68 @@ import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
+import { focusElement } from '../../../common/utils/helpers';
 import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
 import FormTitle from '../../../common/schemaform/components/FormTitle';
 import OMBInfo from '../../../common/components/OMBInfo';
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
   }
   goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
   }
   render() {
     return (
       <div className="schemaform-intro">
-        <FormTitle title="Apply for VR&E"/>
+        <FormTitle title="Apply for vocational rehabilitation for service-disabled Veterans"/>
+        <p>Equal to VA Form 28-1900 (Disabled Veterans Application for Vocational Rehabilitation).</p>
         <SaveInProgressIntro
-          buttonOnly
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
           pageList={this.props.route.pageList}
-          startText="Start the Chapter 31 application"
+          startText="Start the VR&E Application"
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}/>
+        <div className="process schemaform-process schemaform-process-sip">
+          <h4>Follow the steps below to apply for vocational rehabilitiation benefits.</h4>
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>To fill out this application, you’ll need your or your sponsor’s:</h6></div>
+              <ul>
+                <li>Social Security number or VA file number (required)</li>
+                <li>Military status and service history</li>
+                <li>Disability rating</li>
+              </ul>
+              <p><strong>What if I need help filling out my application?</strong><br/>An accredited representative with a Veterans Service Organization (VSO) can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Get help filing your claim</a>.</p>
```

Comment:
This is a little inconsistent with [Chapter 36](https://github.com/department-of-veterans-affairs/vets-website/blob/28481ea65ed19c6b6b2982fb2d0e641618f1a9a2/src/js/vre/chapter36/components/IntroductionPage.jsx#L48). Added a `<br/>` to adhere to the design.

Reply:
Good question, it looks like most of our forms follow the Chapter 36 pattern, if this is a deliberate departure, should those be updated?

Reply:
Perhaps a question for @stacyhlinn ^

Reply:
Probably don't need the `index.html` part in the link.